pkg/sensors/exec/exec_test.go

```diff
@@ -281,6 +281,51 @@ func TestEventExecve(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestEventExecveWithUsername(t *testing.T) {
+	var doneWG, readyWG sync.WaitGroup
+	defer doneWG.Wait()
+
+	ctx, cancel := context.WithTimeout(context.Background(), tus.Conf().CmdWaitTime)
+	defer cancel()
+	mode := cgroups.GetDeploymentMode()
+	ns := namespace.GetCurrentNamespace()
+	if (mode != cgroups.DEPLOY_SD_SERVICE && mode != cgroups.DEPLOY_SD_USER) ||
+		!ns.Mnt.IsHost || !ns.User.IsHost {
+		t.Skip()
```

Comment:
The test is being skipped here: https://github.com/cilium/tetragon/actions/runs/8984334822/job/24676304855?pr=2369#step:6:1349    so you probably need to initialize the deployment as pointed out.

Try to run the test on your local machine with the following command:

```
make
go test -exec "sudo" -p 1 -parallel 1  -gcflags= -timeout 20m -failfast -cover -run TestEventExecveWithUsername ./pkg/sensors/exec/... -v
```

It should not be skipped.